tlsconfig/certpool_other.go

```diff
@@ -0,0 +1,11 @@
+// +build !go1.7
+
+package tlsconfig
+
+import "crypto/x509"
+
+// SystemCertPool returns an new empty cert pool,
+// accessing system cert pool is supported in go 1.7
+func SystemCertPool() *x509.CertPool {
+	return x509.NewCertPool()
```

Comment:
Should we instead be trying to manually read from the default system pool locations to mimic the 1.7 behaviour manually?